jena-arq/src/test/java/org/apache/jena/sparql/core/mem/AbstractTestTupleTable.java

```diff
@@ -42,71 +43,68 @@
 
 	protected static final Set<TupleSlot> allWildcardQuery = of();
 
+	protected long transactionalCount() {
+	    table().begin(READ);
+	    long x = rawCount() ;
```

Comment:
Could this not be
```
table().begin(READ);
try { return rawCount(); }
finally { table().end(); }
```
That seems more readable to me, but maybe that's just my taste.